crates/chain/src/chain.rs

```diff
@@ -807,6 +811,8 @@ impl Chain {
                             .expect("rewind block should exists");
 
                         db.rewind_block_smt(&block)?;
+                        // XXX: delete the bad block?
```

Comment:
Bad blocks are deleted in

- https://github.com/godwokenrises/godwoken/pull/821